index.html

```diff
@@ -2008,35 +2007,35 @@ <h2>The <dfn>record type</dfn> string</h2>
       <td>0</td>
       <td><i>unused</i></td>
       <td>"`empty`"</td>
-      <td>""</td>
+      <td>`null`</td>
```

Comment:
`null` or `undefined`? I'd prefer the latter - but do you have reasons for `null`?

Reply:
`undefined` is fine for me as well.